common/src/sql/dbinit.sql

```diff
@@ -912,6 +912,14 @@ CREATE TABLE omicron.public.role_assignment_builtin (
     PRIMARY KEY(user_builtin_id, resource_type, resource_id, role_name)
 );
 
+/*
+ * Store addresses allocated for internal services (like propolis zones)
+ */
+CREATE TABLE omicron.public.static_v6_address (
+    address STRING(64) PRIMARY KEY NOT NULL,
+    allocated BOOL NOT NULL
```

Comment:
I'm not totally clear on how these addresses are used, but is there a situation in which we want a row in this table, but with the `allocated` column to be `FALSE`? I mean, is there a distinction between a row's existence and `allocated = TRUE`?